cmake/modules/boards.cmake

```diff
@@ -34,6 +34,11 @@
 # CMake module dependencies:
 # - extensions CMake module.
 
+include_guard(GLOBAL)
+
+# Dependencies of this module.
```

Comment:
This patch is using "module" in several places where it should probably be using "cmake module", per @carlescufi 's request to always say either "zephyr module" or "cmake module".

Reply:
fixed at all occurences